src/utils/conversion.ts

```diff
@@ -4947,3 +4963,86 @@ export function confidentialTransactionIdToBigNumber(
 ): BigNumber {
   return new BigNumber(id.toString());
 }
+
+/**
+ * @hidden
+ */
+export function meshPublicKeyToKey(publicKey: PalletConfidentialAssetConfidentialAccount): string {
+  return publicKey.toString();
+}
+
+/**
+ * @hidden
+ */
+export function meshAssetAuditorToAssetAuditors(
+  rawAuditors: BTreeMap<U8aFixed, BTreeSet<PalletConfidentialAssetAuditorAccount>>,
+  context: Context
+): { assets: ConfidentialAsset[]; auditors: ConfidentialAccount[] } {
+  const auditors: ConfidentialAccount[] = [];
+  const assets: ConfidentialAsset[] = [];
+
+  /* istanbul ignore next: nested BTreeMap/BTreeSet is hard to mock */
```

Comment:
the mock would never have `rawAssetAuditors` variable be non iterable, so `...rawAssetAuditors` was not defined, but thats the main way I've been using to iterate on the BTreeSet.